src/tests/frontend/specs/multiple_authors_clear_authorship_colors.js

```diff
@@ -2,36 +2,32 @@
 
 describe('author of pad edition', function () {
   // author 1 creates a new pad with some content (regular lines and lists)
-  before(function (done) {
-    const padId = helper.newPad(() => {
-      // make sure pad has at least 3 lines
-      const $firstLine = helper.padInner$('div').first();
-      $firstLine.html('Hello World');
-
-      // wait for lines to be processed by Etherpad
-      helper.waitFor(() => $firstLine.text() === 'Hello World').done(() => {
-        // Reload pad, to make changes as a second user. Need a timeout here to make sure
-        // all changes were saved before reloading
-        setTimeout(() => {
-          // Expire cookie, so author is changed after reloading the pad.
-          // See https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie#Example_4_Reset_the_previous_cookie
-          helper.padChrome$.document.cookie =
-              'token=foo;expires=Thu, 01 Jan 1970 00:00:00 GMT; path=/';
-
-          helper.newPad(done, padId);
-        }, 1000);
-      });
-    });
+  before(async function () {
     this.timeout(60000);
+    const padId = await helper.aNewPad();
+
+    // make sure pad has at least 3 lines
+    const $firstLine = helper.padInner$('div').first();
+    $firstLine.html('Hello World');
+
+    // wait for lines to be processed by Etherpad
+    await helper.waitForPromise(() => $firstLine.text() === 'Hello World');
+
+    // Need a timeout here to make sure all changes were saved.
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
same here, better use helper.commits, as it is more explicit

Reply:
Done.